Packaging.cmake

```diff
@@ -62,6 +62,21 @@ set(CPACK_DEBIAN_PACKAGE_MAINTAINER "devnull@example.com")
 set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "A tiling window manager")
 set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "/etc/xdg;/usr/share/xsessions")
 
+if(CPACK_GENERATOR MATCHES "RPM")
+    set(CPACK_PACKAGE_FILE_NAME ${CPACK_PACKAGE_NAME})
+    execute_process(
+        COMMAND ${GIT_EXECUTABLE} log -1 --date=format:%Y%m%d --format=%cdgit%h.dirty
+        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+        OUTPUT_VARIABLE GIT_RELEASE_STRING
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
```

Comment:
This only works when building from Git (and git is installed). Should this also be made to work without Git and/or when building from a release tarball (where the version is saved in a file called .version_stamp)?